python/demo/demo_tnt-elements.py

```diff
@@ -67,11 +67,9 @@
 # $f_i\in\operatorname{span}\{q_1, q_2,\dots\}$ (as $\{f_0, f_1,\dots\}$ is a
 # basis of the polynomial space for our element). Using the properties of
 # orthonormal polynomials, we see that
-# \[f_i = \sum_j\left(\int_R f_iq_j\,\mathrm{d}\mathbf{x}\right)q_j,\]
```

Comment:
Not a big deal, but maybe this one and the next one should be `$$` if you still want them to show up in display style

Reply:
I think the current wrapping is fine:)